tests/integration.rs

```diff
@@ -855,8 +855,12 @@ mod tests {
 
     impl<'a> GuestCommand<'a> {
         fn new(guest: &'a Guest) -> Self {
+            Self::new_with_binary_name(guest, "cloud-hypervisor")
+        }
+
+        fn new_with_binary_name(guest: &'a Guest, binary_name: &str) -> Self {
```

Comment:
You could have simply introduced a `new_unprivileged()` function, but either way works for me.